torch/onnx/_internal/exporter/_dispatching.py

```diff
@@ -247,7 +247,7 @@ def get_matching_overload(
                 overload,
                 "The overload does not have a signature. Assuming it is a custom op and matching it.",
             )
-        for param in overload.signature:
+        for param in overload.__pt_signature:  # type: ignore[attr-defined]
```

Comment:
are they different?

Reply:
I’m creating a more private field for this added information, so it will not collide with any public fields we may define on the object in the future

Reply:
Renamed to _pt_onnx_signature to avoid name mangling